### PR TITLE
Modify rule S3699: LaYC format

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3699/javascript/rule.adoc [hunk 1]
```diff
@@ -1,9 +1,15 @@
+This rule raises an issue when a function call results are used however the function does not return anything.
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S3699/javascript/rule.adoc:1]: Reword suggestion:
This rule raises an issue when a function call result is used, even though the function does not return anything.

**Review by guillaume-dequenne-sonarsource — CHANGES_REQUESTED** [review]
Looks good, minor suggestion on the top level description.

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**